.github/workflows/php-compatibility.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - name: Set up PHP
         uses: shivammathur/setup-php@afefcaf556d98dc7896cca380e181decb609ca44 # v2.10.0
         with:
-          php-version: '7.4'
+          php-version: '8.0'
```

Comment:
Please do not use PHP 8.0 yet with PHPCS, at least not until PHPCS has been updated to 3.6.1, and preferably not until WPCS 3.0.0 and PHPCompatibility 10.0.0 have been released.

Reply:
Sorry about that! Was testing this out of curiosity and missed changing this one back in a809d4167f084abd39c2307d448864e60dc3ff46.